tests/onnxruntime/test_modeling.py

```diff
@@ -455,6 +460,63 @@ def test_save_model_with_different_name(self):
 
             self.assertEqual(model.model_name, test_model_name)
 
+    def test_save_ort_model_with_external_data(self):
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            os.environ["FORCE_ONNX_EXTERNAL_DATA"] = "1"  # force exporting small model with external data
+            model = ORTModelForSequenceClassification.from_pretrained(MODEL_NAMES["bert"], from_transformers=True)
+            model.save_pretrained(tmpdirname)
+
+            # verify external data is exported
+            folder_contents = os.listdir(tmpdirname)
+            self.assertTrue(ONNX_WEIGHTS_NAME in folder_contents)
+            self.assertTrue(ONNX_WEIGHTS_NAME + "_data" in folder_contents)
+
+            # verify loading from local folder works
+            model = ORTModelForSequenceClassification.from_pretrained(tmpdirname, from_transformers=False)
+            os.environ.pop("FORCE_ONNX_EXTERNAL_DATA")
+
+    def test_save_decoder_model_with_external_data(self):
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            os.environ["FORCE_ONNX_EXTERNAL_DATA"] = "1"  # force exporting small model with external data
+            model = ORTModelForCausalLM.from_pretrained(MODEL_NAMES["gpt2"], from_transformers=True)
+            model.save_pretrained(tmpdirname)
+
+            # verify external data is exported
+            folder_contents = os.listdir(tmpdirname)
+            self.assertTrue(ONNX_DECODER_NAME in folder_contents)
+            self.assertTrue(ONNX_DECODER_NAME + "_data" in folder_contents)
+            self.assertTrue(ONNX_DECODER_WITH_PAST_NAME in folder_contents)
+            self.assertTrue(ONNX_DECODER_WITH_PAST_NAME + "_data" in folder_contents)
+
+            # verify loading from local folder works
+            model = ORTModelForCausalLM.from_pretrained(tmpdirname, from_transformers=False)
+            os.environ.pop("FORCE_ONNX_EXTERNAL_DATA")
+
+    def test_save_seq2seq_model_with_external_data(self):
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            # randomly intialize large model
+            config = AutoConfig.from_pretrained(self.LARGE_ONNX_SEQ2SEQ_MODEL_ID)
+            with no_init_weights():
+                model = MBartForConditionalGeneration(config)
+
+            # save transformers model to be able to load it with `ORTModel...`
+            model.save_pretrained(tmpdirname)
+
+            model = ORTModelForSeq2SeqLM.from_pretrained(tmpdirname, from_transformers=True)
+            model.save_pretrained(tmpdirname + "/onnx")
+
+            # Verify config and ONNX exported encoder, decoder and decoder with past are present each in their own folder
+            folder_contents = os.listdir(tmpdirname + "/onnx")
+            self.assertTrue(CONFIG_NAME in folder_contents)
+
```

Comment:
This test may slow down the CI a little. We could consider adding `@slow` decorators for such tests

cc @michaelbenayoun @fxmarty @mht-sharma

Reply:
yes!

Reply:
This is the only test failing it seems. It probably fails because it tries to load the whole model and save convert it to external data. We're getting the error
```
worker 'gw0' crashed while running 'tests/onnxruntime/test_modeling.py::ORTModelIntegrationTest::test_save_seq2seq_model_with_external_data'
```
I'm wondering if we should just use a smaller model instead, and use `FORCE_ONNX_EXTERNAL_DATA` ?

Reply:
I think it would be good yes, that way we both test the feature and get a faster (and less prone to OOM) CI.